Source/Tests/LinuxSyscalls/LinuxAllocator.h

```diff
@@ -12,8 +12,9 @@ namespace FEX::HLE {
     virtual void *mmap(void *addr, size_t length, int prot, int flags, int fd, off_t offset) = 0;
     virtual int munmap(void *addr, size_t length) = 0;
     virtual void *mremap(void *old_address, size_t old_size, size_t new_size, int flags, void *new_address) = 0;
-    virtual uint64_t shmat(int shmid, const void* shmaddr, int shmflg, uint32_t *ResultAddress) = 0;
-    virtual uint64_t shmdt(const void* shmaddr) = 0;
+    // Global symbol collision on undercase
```

Comment:
This comment lacks context required to understand it outside of this PR.
```suggestion
    // Named Shmat/Shmdt instead of shmat/shmdt to avoid symbol collision with XYZ
```